### ISSUE TITLE
Removing Windows style paths

### ISSUE THREAD

**Issue by jimmyheller** [MyLibreLab/MyLibreLab#5]
There are some variables in the code with windows style paths that create an exception on *nix environments. It would be better to make them generalized and using the user's home folder to address them. 

the most relevant one is 
```public static String elementPath = "C:\\Users\\velas\\Documents\\NetBeansProjects\\myoepnlab_source\\distribution\\Elements";```

in `FrameMain.java` class which creates an exception in startup.

**Comment by Foadsf** [comment]
@jimmyheller I think this also the reason we get the [`StringOutOfBoundsException ` error](https://github.com/MyLibreLab/MyLibreLab/issues/1), as discussed [here](https://www.reddit.com/r/javahelp/comments/g375tc/javalangstringindexoutofboundsexception_when/fnruiut?utm_source=share&utm_medium=web2x).

**Comment by Mare70** [comment]
Adding more details about the issue:
It happens when the software is run without any arguments. In that case, default element path is:
 `public static String elementPath = "C:\\Users\\velas\\Documents\\NetBeansProjects\\myoepnlab_source\\distribution\\Elements";`

I think there should be a relative path to "distribution/Elements" if the software is run without any args.

**Comment by Mare70** [comment]
@Foadsf Those issues are not connected.

**Comment by LordRibblesdale** [comment]
Hi! I was analysing all Windows references, and I think I might have fixed the first one (elementPath), but I'm not sure how to fix the other ones, like DialogOptions.txtHTMLEditor.text (which refers to "javaeditor.exe").
How should we handle this kind of dependency?

**Comment by weisJ** [comment]
> [...] but I'm not sure how to fix the other ones, like DialogOptions.txtHTMLEditor.text (which refers to "javaeditor.exe").
> How should we handle this kind of dependency?

I’m not quite sure what the `textHTMLEditor` option is for. Is it for external editors? If yes we should either
- provide an in-build editor or
- ask the operating system to open the file with the default program association.

**Comment by LordRibblesdale** [comment]
Yes, it is for external editors. In `"Extras" -> "Options"`, there is "Editors" menu, where you should set your HTML editor.
But it seems that this hypotetical program is not used in the code. There is HTML Editor voice, but nowhere in the code this program is used. `txtHTMLEditor` is a private member of DialogOptions, and no method is used to read this textField.
We could use [Desktop.open(File file)](https://docs.oracle.com/en/java/javase/14/docs/api/java.desktop/java/awt/Desktop.html#open(java.io.File)) when necessary
